ui/apps/platform/src/utils/WorkflowState.js

```diff
@@ -248,6 +248,10 @@ export class WorkflowState {
         return new WorkflowState(useCase, newStateStack, newSearch, newSort, newPaging);
     }
 
+    getStateStack() {
+        return this.stateStack;
+    }
```

Comment:
@alwayshooin , For the 72 release, I'd like you to double-check that Saif and I aren't abusing your WorkflowState. 😈